src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -253,7 +260,60 @@
             })
             .catch(ibexa.helpers.notification.showErrorNotification);
     };
+    const validateInput = (input) => {
+        const isInputValid = !!input.value;
```

Comment:
maybe it would be better with sth like is Input not empty? as non empty input isn't necessary valid, I think